lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -1,112 +1,114 @@
 'use strict';
 
 const BbPromise = require('bluebird');
-const forEach = require('lodash').forEach;
-const merge = require('lodash').merge;
+const _ = require('lodash');
```

Comment:
Looks like Lodash is using this a lot, but imho its more descriptive to call it lodash instead of _